### PR TITLE
Cleanup ocean RPE tests

### PR DESCRIPTION
**Pull request by cbegeman**
This PR includes clean-up of the RPE test cases in the internal wave and baroclinic test groups. It fixes formatted strings and moves the list of viscosities to be tested to the config file.

Checklist
* [X] User's Guide has been updated
* [X] Documentation has been [built locally](https://mpas-dev.github.io/compass/latest/developers_guide/building_docs.html) and changes look as expected

### CODE REVIEW

File: compass/ocean/tests/baroclinic_channel/baroclinic_channel.cfg [hunk 1]
```diff
@@ -48,4 +51,4 @@ gradient_width_dist = 40e3
 salinity = 35.0
 
 # Coriolis parameter for entire domain.
-coriolis_parameter = -1.2e-4
\ No newline at end of file
+coriolis_parameter = -1.2e-4
```
Comment (cbegeman) [inline compass/ocean/tests/baroclinic_channel/baroclinic_channel.cfg:54]: @xylar I haven't seen this before. I'm not sure how to fix it. Does it mean there should be a line at the end of the document?

**Comment by cbegeman** [comment]
This PR makes changes consistent with https://github.com/MPAS-Dev/compass/pull/501 and was prompted by @xylar's review suggestions.

File: compass/ocean/tests/baroclinic_channel/baroclinic_channel.cfg (hunk 1 above)
Comment (xylar) [reply compass/ocean/tests/baroclinic_channel/baroclinic_channel.cfg:54]: The lack of a new line before your changes is undesirable so what you've done here is correct.  Thank you!

**Comment by xylar** [comment]
@cbegeman, I think this PR might also be a good place to fix this docstring:
https://github.com/MPAS-Dev/compass/blob/main/compass/ocean/rpe.py#L9-L23
which should be something like:
```
    Computes the reference (resting) potential energy for the whole domain
    Parameters
    ----------
    initial_state_file_name : str
        Name of the netCDF file containing the initial state

    output_file_prefix : str
        Prefix for the netCDF file containing output of forward step

    num_files : int
        Number of files on which to compute rpe

    Returns
    -------
    rpe : numpy.ndarray
          the reference potential energy of size (num_timesteps) x (num_files)
```

**Comment by cbegeman** [comment]
> @cbegeman, I think this PR might also be a good place to fix this docstring

Done! Thanks for pointing this out.

**Comment by xylar** [comment]
## Testing

I verified that both `ocean/baroclinic_channel/10km/rpe_test` and `ocean/internal_wave/rpe_test` are bit-for-bit compared with `main`.

**Review by xylar — APPROVED** [review]
@cbegeman, thank you for taking the time to do this!

**Pull request merged**